ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -1,22 +1,48 @@
 import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
 
 export default class FormWorkerCreateWorkerLedComponent extends Component {
-  // =attributes
+  // =services
+  @service features;
 
+  // =attributes
   @tracked generatedWorkerAuthToken;
   @tracked clusterId;
   @tracked ipAddress;
   @tracked configFilePath;
   @tracked workerTags;
+  @tracked initialUpstreams;
+
+  // =methods
+
+  textFilter(val) {
```

Comment:
`textFilter` isn't very descriptive, maybe something like `convertCommaSeparatedValuesToArray` or something similar?

Reply:
I do agree with @ZedLi. Can we use a more descriptive name for what this function is doing?